Makefile

```diff
@@ -36,7 +36,7 @@ docs-clean:
 	@cd docs; $(MAKE) clean
 
 lint-flake8:
```

Comment:
Can a comment be added above this line explaining why E501 and F401 are being ignored? Perhaps something like this:

> E501 and F401 are ignored because Pylint performs similar checks.